tsconfig.json

```diff
@@ -7,7 +7,6 @@
     "allowUnreachableCode": false,
     "allowUnusedLabels": false,
     "esModuleInterop": true,
-    "importsNotUsedAsValues": "error",
```

Comment:
why you remove this rule?

Reply:
<img width="772" alt="Screenshot 2024-07-03 at 20 48 22" src="https://github.com/saigontechnology/rn-firebase-chat/assets/114563576/ab0e2993-6fab-4f58-a8e3-7523ba75d754">
This option is deprecated and causing error when compile typescript code
You can read more of this in this link:
https://www.typescriptlang.org/tsconfig/#verbatimModuleSyntax

Reply:
Basically we don't need it anymore, typescript will favor the compiling syntax in the link above